app/models/user.rb

```diff
@@ -0,0 +1,40 @@
+class User < ApplicationRecord
+  attr_accessor :skip_password_validation
+
+  rolify
+
+  extend FriendlyId 
+
+  friendly_id :friendly_field, use: :slugged
+  def friendly_field
+    "#{first_name}-#{last_name}"
+  end
+  
+  has_many :project
+  has_many :lead
+  has_many :leads, through: :lead, source: :project
```

Comment:
this will be confuse to get users, because the name is lead not something related with projects, the same as the pass comment the relation name must be descriptive.